assertion/function/assertiontree/assertion_node.go

```diff
@@ -73,6 +72,10 @@ type assertionNodeCommon struct {
 	parent          AssertionNode // this should be nil for the root
 	children        []AssertionNode
 	consumeTriggers []*annotation.ConsumeTrigger
+
+	// originalExpr stores the original call expression that prompted the creation of this assertion node
```

Comment:
"the original call expression", is this true? 🤔  I think, for example, for field assertion nodes this is a selector expression, right?